pynest/nest/server/hl_api_server.py

```diff
@@ -211,13 +211,11 @@ def do_exec(args, kwargs):
 
     if "return" in kwargs:
         if isinstance(kwargs["return"], list):
```

Comment:
I think you might want to add **tuples** and other ** flat iterable objects **.

```python
from collections.abc import Iterable

def isFlatIterable(obj):
    predicate = lambda obj: isinstance(obj, Iterable) and not isinstance(obj, (str, bytes))
    return predicate(obj) and all(not predicate(item) for item in obj)
```

Reply:
Assuming that this code is used only internally in NEST Server, I do not see the need for this generality, which will come with runtime overhead. So I would leave it as is. What do you think, @babsey?

Reply:
@med-ayssar Thank you for the review. 

I agree @heplesser that it is not necessary to enhance this condition.
`kwargs` comes from the client requests. 

In the documentation we pointed that 'return' can be string.
I was surprised that @vvorobjov founded out the other option.